decksite/data/archetype.py

```diff
@@ -35,8 +35,11 @@ def load_archetype(archetype: Union[int, str], season_id: int = None) -> Archety
         arch.decks = []
     return arch
 
-def load_archetypes(where: str = '1 = 1', merge: bool = False, season_id: int = None) -> List[Archetype]:
-    decks = deck.load_decks(where, season_id=season_id)
+def load_archetypes(where: str = '1 = 1', merge: bool = False, season_id: int = None, tournament_only: bool = False) -> List[Archetype]:
+    having = '1 = 1'
+    if tournament_only:
+        having = 'ct.name != "League"'
```

Comment:
I'd go with `= 'tournament'` over `!= 'League'` although as discussed the meaning is currently the same.

Nit. Our standard quote in SQL is single quote.

Reply:
It looks like the competition_type is named 'Gatherling' - so `= 'Gatherling'` instead of `!= "League"`?